src/helpers/focus_manager.ts

```diff
@@ -0,0 +1,11 @@
+export class FocusManager {
```

Comment:
Should find a better name, I'd expect more of a FocusManagar than to register a single element and expose a method to focus it.

Reply:
I agree, it feels like manager means that we could do "lots" of stuff but I don't have a better idea right know.
`defaultFocus(Service)` maybe?
When you think "manager" what do you expect it to do more? (just fishing for some ideas, I really don't have a clue, and tbh, don't really care as we'll drop this with the store, though the discussion might arise witht the store as well)

Reply:
maybe just `FocusableElement` ?
`env.focusableElement.focus()` feels natural

Reply:
fine with me :)